reactnative/ReactNativeTemplateApp/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'android'
 
 dependencies {
-    compile project(':libs:SmartSync')
+    compile project(':libs:SalesforceSDK')
```

Comment:
All react native apps don't need to use smartstore/smartsync, so we could change forcedroid to ask the user what they want. Thoughts?